src/rpc.cpp

```diff
@@ -133,6 +134,27 @@ RPCServer::RPCServer() {
         RPCObjectID realm_id = store_object(realm_object);
         return (json){{"result", realm_id}};
     };
+    m_requests["/create_user"] = [this](const json dict) {
```

Comment:
Maybe I don't understand the implementation well enough, but if we create the user simply with a `/call_method` request, the object wont be stored in `m_objects`, so we would not be able to access it afterwards? Or are you suggesting that I add that ability to `call_method`?